blocks/bricks/recurrent.py

```diff
@@ -630,3 +631,231 @@ def apply(self, *args, **kwargs):
     @apply.delegate
     def apply_delegate(self):
         return self.children[0].apply
+
+
+class RecurrentStack(BaseRecurrent, Initializable):
+    u"""Stack of recurrent networks.
+
+    Build a stack of recurrent layers from a supplied list of
+    BaseRecurrent objects. Each object must have a `sequences`,
+    `contexts`, `states` and `outputs` parameters to its `apply` method.
```

Comment:
Thanks, this was actually a docstring error, see #702 . `.owner.inputs` should however be not confused with Blocks application method properties, these are inputs of a Theano apply node.